### DISCUSSION TITLE
Which country planned to drop a Nuke on the Moon?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#822] [Galaxy Trivia] [answered]
And When?

**Comment by hasanerzi** [comment]
The United States planned to conduct a nuclear test on the moon as a part of a proposed project called "Project A119" during the Cold War era. The project aimed to demonstrate U.S. capabilities and assert dominance in space exploration. The plan was developed in 1958 by the U.S. Air Force and reportedly received consideration by top-level scientists.

**Comment by Anant-mishra1729** [comment]
Project A119, also known as "A Study of Lunar Research Flights," was a secret United States Air Force plan that aimed to investigate the potential effects of a nuclear explosion on the Moon. The project was conceived in the late 1950s during the Cold War era as a response to the Soviet Union's successful launch of Sputnik.

The goal of Project A119 was to demonstrate American capabilities and assert dominance in space exploration. The plan involved detonating a small nuclear device on the Moon, which would have been visible from Earth and created a dramatic display. However, the project was never carried out, and it remained a theoretical proposal.

Public opposition to the idea of detonating a nuclear weapon in space, concerns about potential negative consequences, and the growing focus on peaceful exploration ultimately led to the cancellation of Project A119. Instead, the U.S. shifted its efforts towards manned missions to the Moon, which resulted in the Apollo program and the successful landing of astronauts on the lunar surface in 1969.

**Comment by peterbaccaro** [comment, marked as answer]
During the Cold War era, there were discussions and proposals within the **United States** regarding the possibility of detonating a nuclear weapon on the Moon as a part of a military demonstration. This project, known as "Project A119," was a top-secret plan **considered by the U.S. Air Force in 1958**. The main purpose of the project was to showcase American technological capabilities and to demonstrate military superiority during the Space Race with the Soviet Union.

However, it is important to note that Project A119 was ultimately abandoned and never executed. Concerns about the potential risks, international backlash, and the negative environmental impact led to the cancellation of the project. The Moon has never been targeted with a nuclear weapon by any country.
